### PR TITLE
Added animation Pranky.css

### PR DESCRIPTION
**Pull request by prankush-tech**

### CODE REVIEW

File: animations/Pranky.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	70% {
+		color: red;
+	}
+
+	100% {
+		color: rgb(55, 16, 10);
+	}
+}
+
+.pranky {
```
Comment (NiallEccles) [inline animations/Pranky.css:22]: Make this uppercase `.Pranky`

File: animations/Pranky.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+	transform-style: preserve-3d;
+	perspective: 600px;
+  animation-name: col;
```
Comment (NiallEccles) [inline animations/Pranky.css:25]: Update this to use your animation name above `pranky`

**Review by NiallEccles — CHANGES_REQUESTED** [review]
Thanks for making this PR.

Can you address the changes I've requested?

File: animations/Pranky.css (hunk 2 above)
Comment (prankush-tech) [reply animations/Pranky.css:25]: Sure I have Updated

**Review by prankush-tech — COMMENTED** [review]
updated

**Pull request merged**